src/explore-education-statistics-admin/src/pages/admin-dashboard/components/__tests__/ApprovalsTable.test.tsx

```diff
@@ -0,0 +1,186 @@
+import ApprovalsTable from '@admin/pages/admin-dashboard/components/ApprovalsTable';
+import { MethodologyVersion } from '@admin/services/methodologyService';
+import { Release } from '@admin/services/releaseService';
+import { waitFor, within } from '@testing-library/dom';
+import { render, screen } from '@testing-library/react';
+import React from 'react';
+import { MemoryRouter } from 'react-router';
+
+describe('ApprovalsTable', () => {
+  const testMethodologies: MethodologyVersion[] = [
+    {
+      id: 'c8c911e3-39c1-452b-801f-25bb79d1deb7',
+      methodologyId: 'b8bd000c-f9d8-4319-a2b3-6bc18675e5ac',
+      owningPublication: {
+        id: 'bf2b4284-6b84-46b0-aaaa-a2e0a23be2a9',
+        title: 'Permanent and fixed-period exclusions in England',
+      },
+      otherPublications: [],
+      published: '2018-08-25T00:00:00',
+      publishingStrategy: 'Immediately',
+      slug: 'permanent-and-fixed-period-exclusions-in-england',
+      status: 'Approved',
+      title: 'Pupil exclusion statistics: methodology',
+      amendment: false,
+    },
+  ];
+
+  const testReleases: Release[] = [
```

Comment:
A lot of the extra cruft can be removed from this test data if we get the list release endpoints to return `ReleaseSummary` with a new optional `publication` property.

I've mentioned this in another comment on the backend.